package.json

```diff
@@ -29,7 +29,8 @@
     "nocache": "^2.1.0",
     "node-pty": "^0.10.1",
     "range-stream": "^1.1.0",
-    "send-seekable": "^1.0.4"
+    "send-seekable": "^1.0.4",
+    "vue-router": "^4.2.5"
```

Comment:
I ended up added routing because
1) this felt cleaner. The alternative was to add a <LandingPage /> component to the App.vue file and then programmatically check if the store `page === 'landingpage'`. If `page === 'landingpage'` then I'd have to hide the <aside /> and programmatically remove the <aside /> related CSS.
2) we probably want routing in the future. Someone should be able to visit the URL and continue editing from where they left off, or they should be able to send their draft to someone else. In order to do this, we would want some sort of ezBIDS url like `https://brainlife.io/ezbids/subject#sessionid=abc123` or `https://brainlife.io/ezbids/finalize#sessionid=abc123`